app/models/tracer.rb

```diff
@@ -21,7 +23,13 @@
 class Tracer < ApplicationRecord
   include Concerns::TracerManager
 
-  has_attached_file :photo
+  has_attached_file :photo,
```

Comment:
Does it imply to install Image Magic or something like that?

Reply:
Image Magick yes
https://github.com/thoughtbot/paperclip/wiki/Thumbnail-Generation#resizing-options

Reply:
Ok, I added it to the new build workflow https://github.com/FranckKe/mersea/pull/140/files#diff-3254677a7917c6c01f55212f86c57fbfR69